### PR TITLE
Fastag details and Vehicle details have been added into the transporter app

### PR DESCRIPTION
**Pull request by Vedantfuley**
Fastag APIs and Vahan APIs have been integrated into the Transporter App.

### CODE REVIEW

File: lib/screens/TransporterOrders/documentUploadScreen.dart [hunk 1]
```diff
@@ -68,8 +73,9 @@ class _documentUploadScreenState extends State<documentUploadScreen> {
   @override
   void initState() {
     super.initState();
+    print("hiii");
```
Comment (Anik-Chakraborty) [inline lib/screens/TransporterOrders/documentUploadScreen.dart:76]: remove such print statements

File: lib/screens/TransporterOrders/documentUploadScreen.dart [hunk 2]
```diff
@@ -1442,7 +1453,32 @@ class _documentUploadScreenState extends State<documentUploadScreen> {
                                       ),
                                     ),
                                   ),
-
+                                  // Padding(
+                                  //   padding: EdgeInsets.fromLTRB(
+                                  //   space_4, space_4, space_4, 0,),
```
Comment (Anik-Chakraborty) [inline lib/screens/TransporterOrders/documentUploadScreen.dart:1458]: remove this commented code.

File: pubspec.lock
Comment (Anik-Chakraborty) [inline pubspec.lock:1]: do not send pubspec.lock file with pr... it is a autogenerated file no need to add this.
Rather it spoiled your credentials like sha keys.

File: lib/screens/TransporterOrders/documentUploadScreen.dart [hunk 3]
```diff
@@ -15,6 +15,9 @@ import 'package:liveasy/screens/TransporterOrders/navigateToTrackScreen.dart';
 //import 'package:liveasy/screens/TransporterOrders/putDocumentApiCall.dart';
 import 'package:permission_handler/permission_handler.dart';
 import 'package:provider/provider.dart';
+import '../../widgets/buttons/fastagButton.dart';
+// import '../../widgets/buttons/sarathiButton.dart';
```
Comment (Anik-Chakraborty) [inline lib/screens/TransporterOrders/documentUploadScreen.dart:19]: remove this one

File: pubspec.lock
Comment (Vedantfuley) [reply pubspec.lock:1]: ok

**Review by Anik-Chakraborty — APPROVED** [review]

File: lib/screens/fastagScreen.dart [hunk 4]
```diff
@@ -0,0 +1,687 @@
...
+                                  child: Text(
+                                    "${widget.bookingDate}",
+                                    style: TextStyle(
+                                        color: white, fontSize: size_6),
+                                  ),
+                                ),
+                                Padding(
+                                  padding: const EdgeInsets.only(top: 3),
+                                  child: Text(
+                                    "8:47 AM",
```
Comment (chirag4601) [inline lib/screens/fastagScreen.dart:624]: why is this time hardcoded?

Comment (Vedantfuley) [reply lib/screens/fastagScreen.dart:624]: We don't have a variable for displaying the time for booking right now. For solving this issue i will have to create variable for this in model class then display it into the screen dynamically.

**Review by chirag4601 — APPROVED** [review]

**Pull request merged**